tests/test_cases/test_axi4/test_axi4.py

```diff
@@ -521,7 +532,9 @@ class AXI4Master_unmanaged_arlen(AXI4Master):
 incr_burst_size = TestFactory(test_incr_burst)
 incr_burst_size.add_option('return_rresp', (False,))
 incr_burst_size.add_option('size', (1, 2))
-incr_burst_size.generate_tests(postfix="_size")
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    incr_burst_size.generate_tests(postfix="_size")
```

Comment:
These should be updated to be cocotb.parametrize at some point.

Reply:
Unfortunately cocotb.parametrize is not available in 1.9.0. As long as 1.x is supported by cocotb-bus we'll need to use older functionality.